examples/buttons/index.html

```diff
@@ -90,23 +96,70 @@ <h2 id="button with icon"><a href="#button-with-icon">button with icon</a></h2>
       )
     })
   </script>
-  <pre id="viewSource-button-with-icon"></pre><script>showSource('button-with-icon')</script>
+  <pre id="viewSource-button-with-icon"></pre>
+  <script>showSource('button-with-icon')</script>
   <div id="div-button-with-icon"></div>
 
- <h2 id="button-with-text"><a href="#button-with-text">button with text</a></h2>
-  <script id="script-button-with-text">
```

Comment:
I have added all the button options here in the examples, but the secondary isn't working right now due to the enum..  I will fix tomorrow.